Modules/UpendoDnnSimpleAuthProvider/Login.ascx.cs

```diff
@@ -482,6 +479,9 @@ protected void btnSendEmail_Click(object sender, EventArgs e)
             // Check if the user object (objUser) is not null
             if (objUser != null)
             {
+                // Get the user's name from the user object
+                string username = objUser.Username;
+
                 // Get the user's email address from the user object
                 string emailAddress = objUser.Email;
```

Comment:
@ArielBlanco1990 Is this value being used anymore?  If not, it can be deleted, right?

Reply:
@WillStrohl It's currently being used, though you could take the username value  directly from objUser.Username and remove this variable.

Reply:
@ArielBlanco1990 I was intending to ask that about the `.Email` property.  Is that still in use?  If not, can it be removed?

Reply:
I am getting the Email property from the email associated with the user to send the email with the verification code.  At startup you just need to type your Username, but I need to send the mail to the email account associated with that Username.

Reply:
@WillStrohl Any questions you have are welcome.  A pleasure to answer each of them. :)